corehq/apps/case_importer/templates/case_importer/partials/ko_import_status.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+<!--ko if: state === $root.states.SUCCESS-->
+    <div class="text-success">
+        <!--ko if: result.match_count-->
+            <p>
+                {% blocktrans with match_count='result.match_count' %}
+                    <strong data-bind="text: {{ match_count }}"></strong> rows were matched and
+                    updated.
+                {% endblocktrans %}
+            </p>
+        <!--/ko-->
+
+        <!--ko if: result.created_count > 0-->
+            <p>
+                {% blocktrans with created_count='result.created_count' %}
+                    <strong data-bind="text: {{ created_count }}"></strong> rows did
```

Comment:
Also confused by this, why use Knockout data-bind with a static value instead of just putting the value in?